app/pb_kits/playbook/pb_badge/_badge.jsx

```diff
@@ -32,12 +32,8 @@ const Badge = (props: BadgeProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const css = classnames(
-    className,
-    buildCss('pb_badge_kit', variant, {
-      rounded: rounded,
-    }),
-    globalProps(props)
-  )
+    buildCss('pb_badge_kit', variant, { rounded: rounded }), className,
+    globalProps(props))
```

Comment:
```suggestion
    buildCss('pb_badge_kit', variant, { rounded: rounded }), globalProps(props), className)
```

Reference https://github.com/powerhome/playbook/pull/929/files